### PR TITLE
Composer init

### PR DESCRIPTION
**Pull request by luigimassa**
Added composer setup to enable service

### CODE REVIEW

File: src/content/1.7/modules/concepts/commands.md [hunk 1]
```diff
@@ -22,6 +22,39 @@ You could rely on the webservices, but they are not really easy to configure. Th
 
 You need to create the file and register it as a "command".
 
+### Setup composer
+
+You need setup composer in your module before create the command.
+Create the file `your-module/composner.json` and paste:
```
Comment (PierreRambaud) [inline src/content/1.7/modules/concepts/commands.md]: `composer.json` => `composner.json`

File: src/content/1.7/modules/concepts/commands.md [hunk 2]
```diff
@@ -22,6 +22,39 @@ You could rely on the webservices, but they are not really easy to configure. Th
 
 You need to create the file and register it as a "command".
 
+### Setup composer
+
+You need setup composer in your module before create the command.
+Create the file `your-module/composner.json` and paste:  
+```json
+{
+    "name": "<your name>/<nmodule name>",
```
Comment (PierreRambaud) [inline src/content/1.7/modules/concepts/commands.md]: `module name` => `nmodule name`

File: src/content/1.7/modules/concepts/commands.md [hunk 3]
```diff
@@ -22,6 +22,39 @@ You could rely on the webservices, but they are not really easy to configure. Th
...
+    "description": "<module description>",
+    "authors": [
+        {
+            "name": "<your name>",
+            "email": "<your email>"
+        }
+    ],
+    "require": {
+        "php": ">=7.1.0"
```
Comment (PierreRambaud) [inline src/content/1.7/modules/concepts/commands.md]: We could recommend php 5.6 to be compliant with PrestaShop composer.json?

File: src/content/1.7/modules/concepts/commands.md [hunk 4]
```diff
@@ -22,6 +22,39 @@ You could rely on the webservices, but they are not really easy to configure. Th
...
+    },
+    "config": {
+        "preferred-install": "dist"
+    },
+    "type": "prestashop-module",
+    "author": "<???>",
+    "license": "<???>"
+}
+```
+In __YourModule__ add your name space. Then in console in your module root run command `composer dump-autoload`.
```
Comment (PierreRambaud) [inline src/content/1.7/modules/concepts/commands.md]: namespace in one word.

File: src/content/1.7/modules/concepts/commands.md (hunk 1 above)
Comment (luigimassa) [reply src/content/1.7/modules/concepts/commands.md]: ok

**Comment by mickaelandrieu** [comment]
Thanks @luigimassa and @PierreRambaud for the review

**Pull request merged**